packages/libelf/debuginfod-client.c.patch

```diff
@@ -0,0 +1,23 @@
+--- elfutils-0.185/debuginfod/debuginfod-client.c	2021-05-22 23:55:24.000000000 +0530
++++ elfutils-0.185-patch/debuginfod/debuginfod-client.c	2021-08-28 22:14:21.559999738 +0530
+@@ -85,6 +85,20 @@
+ #include <sys/utsname.h>
+ #include <curl/curl.h>
+ 
++int futimes(int fd, const struct timeval tv[2]) {
```

Comment:
Why not to declare it with keyword `static`?

Reply:
Oh yes....updating it.